Signup/Signup.css

```diff
@@ -0,0 +1,59 @@
+@import url(https://king-ui.netlify.app/styles.css);
+@import url(https://king-ui.netlify.app/Stylesheets/navbar.css);
+
+@import url(https://king-ui.netlify.app/Stylesheets/reset.css);
```

Comment:
I suggest use only single link for your CL import

Reply:
noted